clio-util/src/main/scala/org/broadinstitute/clio/util/model/EntityId.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.clio.util.model
+
+object EntityId {
+
+  /**
+    * Clio name for an entity ID.
+    */
+  val EntityIdFieldName: String = "entityId"
+}
```

Comment:
Having a separate object just for this seems like overkill.

Looking ahead, I think it (and the method to get the entity ID out of a JSON blob) belongs in `ElasticsearchIndex`, because after [DSDEGP-2121](https://broadinstitute.atlassian.net/browse/DSDEGP-2121) the hard-coded `entity_id` field won't exist any more and we'll need to be able to vary our ID-calculation logic on an index-by-index basis.

Reply:
We can also go straight to snake case for the field name now that it's not part of any case-classes.